src/main/java/tr/com/srdc/cda2fhir/transform/entry/IEntryResult.java

```diff
@@ -1,22 +1,14 @@
 package tr.com.srdc.cda2fhir.transform.entry;
 
-import java.util.List;
-
 import org.hl7.fhir.dstu3.model.Bundle;
 
-import tr.com.srdc.cda2fhir.transform.util.ICDAIIMapSource;
-import tr.com.srdc.cda2fhir.transform.util.IDeferredReference;
+import tr.com.srdc.cda2fhir.transform.util.IResult;
 
-public interface IEntryResult extends ICDAIIMapSource<IEntityInfo> {
+public interface IEntryResult extends IResult {
```

Comment:
Created a shared interface for IEntryResult & ISectionResult.

Reply:
(Necessary because I needed an .updateFrom(ICDAIIMapSource<IEntityInfo>) and updateFrom(ICDAIIMapSource<IResultsMap>) in bundleInfo, but you can't overload with generic types. Solved it with an interface that extends both (IResult).